cmd/chaos-daemon/main.go

```diff
@@ -50,21 +51,13 @@ func main() {
 
 	ctrl.SetLogger(zap.Logger(true))
 
-	if rawPort == "" {
-		rawPort = os.Getenv("PORT")
```

Comment:
This environment variable is set in the helm script. If you want to remove this environment variable, please also update the helm script https://github.com/pingcap/chaos-mesh/blob/master/helm/chaos-mesh/templates/chaos-daemon-deployment.yaml#L41

Reply:
Done